src/value/native_reader.rs

```diff
@@ -26,23 +26,9 @@ impl ElementReader for NativeElementReader {
         &'a self,
         data: &'b [u8],
     ) -> IonResult<Box<dyn Iterator<Item = IonResult<OwnedElement>> + 'b>> {
-        // Match against the slice to see if it starts with the binary IVM
```

Comment:
🗺️ This code was moved to the body of `ReaderBuilder::build` earlier in the diff. (It would probably make sense to offer it as a utility method, but I'll take care of that in a future PR.)